twitter_clone/main.py

```diff
@@ -27,3 +27,106 @@ def decorated_function(*args, **kwargs):
 
 
 # implement your views here
+@app.route('/login', methods=['POST', 'GET'])
+def login():
+	if request.method == 'GET':
+		if session.get('username', None) is not None:
```

Comment:
we did `if 'username' in session:`. am wondering what the difference is between our code and `if session.get('username', None) is not None` - not sure whether you'd know but thought I'd ask just in case since you're half of the canadian power team.